module-1/src/Components/MovieForm/MovieForm.tsx

```diff
@@ -1,16 +1,116 @@
+import { ChangeEvent, useState } from "react";
+import { SubmitHandler, useForm } from "react-hook-form";
+import { toast } from "react-toastify";
+import { TestData } from "../../TestData";
 import "./MovieForm.css";
 interface MovieFormProps {
   title: string;
   movie?: Movie;
   closeCallback: () => void;
 }
 
+type FormData = {
+  id: number;
+  title: string;
+  tagline?: string;
+  vote_average: number;
```

Comment:
Use lowerCamelCase for variable in JS/TS, it's the industry standard. If some API response prop names don't alight with it, map it to your own type on fetch.

Reply:
fixed